### PR TITLE
ReversingLabs RLSDK 1.3.0 updates

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Updates the apps with ReveringLabs SDK 1.3.0 functions while not breaking backward compatibility.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by MislavReversingLabs** [comment]
Hi! I did not attach any video to the contribution form because everything stayed completely the same from a user's perspective. The only changes made were the related to implementing the latest version of our ReversingLabs SDK.
For additional questions and corrections, contact my colleague @ivukovicRL

**Comment by MislavReversingLabs** [comment]
Can you also add @ivukovicRL as a participant in this pull request so he can receive emails and notifications from you and CircleCI?

File: Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/ReversingLabsTitaniumScale.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import requests
 import json
 
-VERSION = "v1.0.2"
+VERSION = "v1.0.3"
```
Comment (omerKarkKatz) [inline Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/ReversingLabsTitaniumScale.py:8]: Does the version change have no effect on the code?

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
     RHA1Analytics, URIStatistics, URIIndex, AdvancedSearch, ExpressionSearch, FileDownload, FileUpload, \
     URLThreatIntelligence, AnalyzeURL, DynamicAnalysis, CertificateAnalytics
 
-VERSION = "v2.0.4"
+VERSION = "v2.0.5"
```
Comment (omerKarkKatz) [inline Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py:8]: Does the version change have no effect on the code?

**Review by omerKarkKatz — COMMENTED** [review]
Hey @MislavReversingLabs, thanks a lot for the contribution looks good! 
I have some suggestions and questions on this PR. 
Please look at the unit tests covering this code looks like they are affected by your changes and some of them fail please fix the unit tests or the changes.
Also are you sure that the results returning after the bump of the SDK version are exactly the same? otherwise it will be better to state in the release notes, the places where the backward compatibility is affected. As I understand now there is no effect at all for users already using this pack. 
backward compatibility counts for any change in the data returned to the context of XSOAR key or value. and also readable outputs returned to the user on the WarRoom. 
If you have any questions or you need any help please feel free to reach out here or via slack. 
Besides that, before I merge this PR we need to do a short demo to cover the changes so when you finish fixing the above pleases write me when will be a good time for you for a demo.
Thanks a lot @omerKarkKatz

File: Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/ReversingLabsTitaniumScale.py (hunk 1 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/ReversingLabsTitaniumScale.py:8]: For the TitaniumScale and TitaniumCloud apps it doesnt. In the case of these two apps, only the Docker image was changed.

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py (hunk 2 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py:8]: Same answer as above.

**Comment by omerKarkKatz** [comment]
@MislavReversingLabs https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version.
As we discussed a guide to documenting breaking changes in XSOAR.

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_0_8.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":null}
```
Comment (omerKarkKatz) [inline Packs/ReversingLabs_A1000/ReleaseNotes/2_0_8.json:1]: Please add the note here

**Comment by omerKarkKatz** [comment]
@MislavReversingLabs Thanks for the modifications looks good! 
Please add some notes on the breaking changes .json file, with explanations, then I can merge your PR.

**Comment by omerKarkKatz** [comment]
BTW is there a reason you decided to delete all the doc_imgs ? @MislavReversingLabs

**Comment by MislavReversingLabs** [comment]
> BTW is there a reason you decided to delete all the doc_imgs ? @MislavReversingLabs

@omerKarkKatz Yes, I did it because we're no longer displaying human readable context in the readme via images. Now we have acutal markdown sections for that. It's created via demisto-sdk.
Also, one question: How am I supposed to ad notes about breaking changes to the json file? Isn't the markdown file for notes about changes?

**Comment by omerKarkKatz** [comment]
@MislavReversingLabs This is the message that will be presented to the customers when updating your pack. 
It can be a more general message like updating the SDK version causing some changes in human readable and context outputs. for more information look at the release notes.

**Comment by MislavReversingLabs** [comment]
@omerKarkKatz Ok, done. Can you please check the PR and merge it?

**Review by omerKarkKatz — APPROVED** [review]

**Pull request merged**